src/content_script.js

```diff
@@ -68,6 +68,13 @@ const PASSIVE_SHARE_PATTERN_DETECTION_SELECTORS = [
   "[href*='facebook.com/sharer']", // Legacy Share dialog
 ];
 
+// Attributes distilled from selectors above. Update when necessary.
+const OBSERVER_ATTRIBUTES = [
+  "action", "aria-label", "class",
```

Comment:
Commenting here, even though the PR is closed. 

Is there a way to generically watch any data-attributes that contain "Facebook", or "FB", rather than having to list each attribute to watch?

Reply:
Hmm good question. I think you'd have to omit the attributeFilter entirely and filter manually. This would mean that the MutationObserver will get notified for things like animations that use the style attribute, so the code to reject that mutation would need to be really fast.

I could try to match with a regex instead and if a match is found inspect the attribute value to match on "facebook" or "fb".

Reply:
Let's not over optimize – how you have it works great for now. There are some legacy / super-site specific artifacts in here that can be eliminated, but we can worry about that later too.